src/cart/SConscript

```diff
@@ -141,7 +141,8 @@ def scons():
     Export('env', 'prereqs')
 
     # generate targets in specific build dir to avoid polluting the source code
```

Comment:
Remove these lines if they are not needed

Reply:
aaa64142e6c6865e6d2f0b090f93acfa182ba830